src/main/java/org/folio/circulation/resources/MarkItemMissingResource.java

```diff
@@ -0,0 +1,55 @@
+package org.folio.circulation.resources;
+
+import static org.folio.circulation.domain.validation.NotInItemStatusValidator.refuseWhenItemIsNotClaimedReturned;
+import static org.folio.circulation.support.Result.failed;
+import static org.folio.circulation.support.Result.succeeded;
+import static org.folio.circulation.support.ValidationErrorFailure.singleValidationError;
+
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.domain.representations.ChangeItemStatusRequest;
+import org.folio.circulation.support.Result;
+
+import java.util.concurrent.CompletableFuture;
+
+public class MarkItemMissingResource extends ChangeStatusResource {
+  public static final String COMMENT = "comment";
```

Comment:
Can it be private? 

```suggestion
  private static final String COMMENT = "comment";
```

Reply:
I've removed this field as there is the same field in ChangeStatusResource.  The constant can't be final as it is used in some tests.